### ISSUE TITLE
Wilderness & Mages - Mages acquiring wilderness lands such as c_rhosneiger and becoming independent.

### ISSUE THREAD

**Issue by ajjr1996** [ajjr1996/Witcher#112]

**Comment by rquinio** [comment]
Is it the same issue as https://github.com/JonStryker/Witcher/issues/65 ? 
If you can check your game.log and title history to understand what happened that would help.

**Comment by ajjr1996** [comment]
It doesn't appear to have any info in the history file. I'll try to check the game.log

**Comment by ajjr1996** [comment]
It's similar to #65 but it appears that the mage isn't employed this time or a vassal of d_wilderness, it's instead independent now.

**Comment by ajjr1996** [comment]
I noticed it will keep trying to give out vassals to other characters with the wilderness culture and then take it back and repeat over and over.

**Comment by rquinio** [comment]
I don't think so. I tried making the wilderness an offmap character, but the game doesn't like it owning landed titles then and it crashes.

**Comment by ajjr1996** [comment]
Strange.. I’m not certain how the wilderness works. Unfortunately it just makes it hard to play. I wonder if Jonstryker would know anything?

**Comment by ajjr1996** [comment]
Looking good so far this play through, I've noticed people are having children with no mother. Wilderness seems to have a son from this lol

**Comment by rquinio** [comment]
Few things should now be fixed, let me know if you see anything suspicious.

**Comment by ajjr1996** [comment]
Sounds good! I’ll test it out as soon as I get home.

**Comment by ajjr1996** [comment]
Sometimes I'll also notice there are characters that are of the wilderness culture that are humans, so far it's working pretty good though!
